File: src/JavaScript/easy/string/problem2185.js
/**
 * @param {string[]} words
 * @param {string} pref
 * @return {number}
 */

// First solution

var prefixCount = function(words, pref) {
    let i = 0
    words.forEach(word => {
        if(word.indexOf(pref) == 0){
            i++
        }
    })
    return i
};


// Second solution

var prefixCount = function(words, pref) {
    let i = 0
    words.forEach(word => {
        if(word.startsWith(pref)){
            i++
        }
    })
    return i
};

// let words = ["pay","attention","practice","attend"], pref = "at"
let words = ["leetcode","win","loops","success"], pref = "code"

prefixCount(words, pref)

File: src/Python/easy/string/problem2185.py
from typing import List

class Solution:
    def prefixCount(self, words: List[str], pref: str) -> int:
        i = 0
        for word in words:
            if word.startswith(pref):
                i += 1
        return i

words = ["pay","attention","practice","attend"]; pref = "at"

solution = Solution()
solution.prefixCount(words, pref)